src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java

```diff
@@ -404,14 +404,13 @@ protected ImmutableList<String> createArgs() {
     }
 
     @Override
-    public AppiumDriverLocalService build() {
```

Comment:
was the `build` method removed?

Reply:
I would say it is ok to have loadSystemProperties.
Although I assume the "original" build method returns is typed by the class, which is probably not what we want here. That is why it still makes sense to override it in order to better reflect the returned instance type

Reply:
> Although I assume the "original" build method return is typed by the class, which is probably not what we want here (the parent one). That is why it still makes sense to override it in order to better reflect the returned instance type

Actually it uses generics and the returned type is specified by Builder declaration:
https://github.com/appium/java-client/blob/f2fb595a81cc8cfeca22fd7ff80847f277c2a2a9/src/main/java/io/appium/java_client/service/local/AppiumServiceBuilder.java#L58-L59
so it works perfectly for Appium and the result of `build` method is of `AppiumDriverLocalService` type.